### PR TITLE
update to allow RELEASE_TAG to be set by noironetworks/cicd

### PR DESCRIPTION
**Pull request by anmol372**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,7 +34,7 @@ jobs:
         - /tmp/cicd/travis/check-git-tag.sh; RETURN_CODE=$? ; if [ $RETURN_CODE -eq 140 ]; then travis_terminate 0; elif [ $RETURN_CODE -ne 0 ]; then travis_terminate $RETURN_CODE; fi
         - pip install boto3 || travis_terminate 1
         - python provision/setup.py install || travis_terminate 1
-        - git clone --depth=50 --branch=${RELEASE_TAG} https://github.com/noironetworks/aci-containers.git /tmp/noironetworks/aci-containers
+        - /tmp/cicd/travis/clone-aci-containers-repo.sh || travis_terminate 1
```
Comment (snaiksat) [inline .travis.yml:37]: Is this change intended?

Comment (anmol372) [reply .travis.yml:37]: yes, cicd includes the script to run the above command: https://github.com/noironetworks/cicd/pull/24/files

**Review by abhis2112 — APPROVED** [review]
